src/coreclr/debug/daccess/cdac.cpp

```diff
@@ -48,7 +48,7 @@ CDAC::CDAC(HMODULE module, uint64_t descriptorAddr, ICorDebugDataTarget* target)
 {
     if (m_module == NULL)
     {
-        m_cdac_handle = NULL;
+        m_cdac_handle = 0;
```

Comment:
looks like the approach in https://github.com/dotnet/runtime/pull/100796 was to cast NULL to INT_PTR instead

Reply:
I think it's a style preference.  in this case m_cdac_handle isn't a pointer (it happens to be a NativeAOT GCHandle), so `(INT_PTR)NULL` isn't really apropriate, either.